docs/site/src/components/GraphqlBetaLink/index.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useEffect, useState } from "react";
+import { useLocation } from "@docusaurus/router";
+import Link from "@docusaurus/Link";
+
+export default function GraphqlBetaLink({ title }) {
+  /*const { pathname } = useLocation();
+  const [betaExists, setBetaExists] = useState(false);
+
+  const betaPath = pathname.replace("/alpha/", "/beta/");
+
+  useEffect(() => {
+    if (pathname.includes("/alpha/")) {
+      fetch(betaPath, { method: "HEAD" })
+        .then((res) => {
+          if (res.ok) setBetaExists(true);
+        })
+        .catch(() => {});
+    }
+  }, [pathname, betaPath]);
+  */
+  return (
+    <div className="bg-yellow-100 text-yellow-900 p-4 rounded mb-6 text-center border border-yellow-300">
+      <>
+        ⚠️ This is the <strong className="mx-1">beta</strong> version of the Sui
+        GraphQL schema. The beta schema will replace the{" "}
+        <Link href="/references/sui-api/sui-graphql/alpha/reference/operations/directives/include">
```

Comment:
This link is a bit arbitrary (it just happens to be the first leaf node in the docs hierarchy but it's not significant in any way), the issue is that we don't have a landing page for the overall schema. Maybe we could add one, and link to that? Otherwise I would suggest linking to `references/sui-graphql#graphql-schemas`